docs/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-icon: octicons/beaker-24
+icon: kolena/test-suite-16
```

Comment:
Is it intentional to use the same icon for test-suite for `getting-started` in the side-bar?

Reply:
👍 would be good to use a different icon

Reply:
Will address in a follow-up